lib/components/sitewide/TitleBar.vue

```diff
@@ -72,14 +74,14 @@ v-toolbar(app)
     },
 
     computed: {
-      ...mapGetters([ "loggedIn", "showSpinner", "isProduction" ]),
+      ...mapGetters([ "loggedIn", "showSpinner", "isProduction", "darkMode" ]),
 
       homeLink () {
         return this.loggedIn ? 'gameManager' : 'splash'
       }
     },
 
-    methods: mapActions([ "saveToDrive" ])
+    methods: {...mapActions([ "saveToDrive", "setDarkMode" ])}
```

Comment:
again unnecessary to add the `{... }` because there is nothing else being added to `methods`, so this is cloning an object for no benefit.